src/css/style.css

```diff
@@ -0,0 +1,88 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  font-family: "Roboto", Helvetica, sans-serif;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 200px;
+  border-radius: 5px;
+  background-color: var(--color-white);
+  border: 1px solid var(--border-color);
+  color: var(--color-dark);
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0 16px 4px;
+}
+
+.card__description {
+  font-size: 10px;
+  margin-left: 16px;
```

Comment:
try to use margin top or bottom only
<img width="704" alt="CleanShot 2022-08-21 at 16 20 01@2x" src="https://user-images.githubusercontent.com/103505670/185792872-1c23b8b9-e3c9-4e9a-97bd-ff95e969f938.png">

Reply:
ok